optuna/logging.py

```diff
@@ -102,6 +102,31 @@ def get_verbosity() -> int:
 def set_verbosity(verbosity: int) -> None:
     """Set the level for the Optuna's root logger.
 
+    Example:
+
+        Set the logging level ``optuna.logging.INFO``.
+
+        .. testsetup::
+
+            def objective(trial):
+                x = trial.suggest_int("x", -10, 10)
+                return x ** 2
+
+        .. testcode::
+
+            import optuna
+
+            # set the logging level
```

Comment:
To be consistent with other docstring examples, please make the sentence starting with the capital letter and end with period.